.github/workflows/build.yml

```diff
@@ -173,3 +172,19 @@ jobs:
           chmod +x `pwd`/target/release/spin
           export E2E_VOLUME_MOUNT="-v `pwd`/target/release/spin:/usr/local/bin/spin"
           make test-spin-up
+
+  lint-examples:
+    name: Lint Examples
```

Comment:
Should we try to break these changes out to a separate PR? Seems useful and it would be a shame to block these changes on landing `wasi-http` support (which isn't really related).